js/panel-manager.js

```diff
@@ -0,0 +1,57 @@
+// var count = 0;
+
+var PanelManager = {
+//$(document).ready(function(){
+//  $('#btn-add').click(function(e){
+  init: function(e){
+    $('.panel-manager').load('templates/panel-manager.html', function(e){
+      $('#slidebar').on('click', function(e){
+        $('#slider').toggle('slide', { direction: 'right' }, 300);
+        $('#slidebar').animate({
+          'right' : $('#slidebar').css('right') == '0px' ? '249px' : '0px'
+        }, 300);
+        setTimeout(function(){
+          $('#icon-button').toggleClass('glyphicon-chevron-right glyphicon-chevron-left');
+        }, 300);
+      });
+    });
+  },
+
+  addPanel: function(view){
+//     count++;
+    var count = $('.sideways').children().length;
+//    e.preventDefault();
+//    $('.sideways').append('<li id="li'+count+'"><a href="#view'+count+'" data-toggle="tab">view'+count+'</a></li>');
+//    $('.sideways').append('<li id="li'+count+'"><a href="#view'+count+'" data-toggle="tab">'+viewName+'</a></li>');
+    var viewID = view.name().replace(/\s/g, '');
+    $('.sideways').append('<li id="li'+viewID+'"><a href="#'+viewID+'" data-toggle="tab">'+view.name()+'</a></li>');
```

Comment:
It's a bit nasty to hard code DOM with jquery like this (sorry about my bad previous practice! :P ). Is there a better way to add DOM element? 

If not, please format the line a bit. In JS, we add a single space between the operands, e.g.
'string text...' + viewID + 'string text...'.
You may use multiple lines when there are string concats.